### PR TITLE
Add visible on hit for algo order

### PR DESCRIPTION
**Pull request by avsek477**
Task: https://app.asana.com/0/1125859137800433/1201518274330313

Depends on: 

- https://github.com/bitfinexcom/bfx-api-node-models/pull/77
- https://github.com/bitfinexcom/bfx-api-node-core/pull/29

Add visible on hit option for hidden orders placed by algo orders.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
   "dependencies": {
     "@bitfinex/lib-js-util-math": "git+https://github.com/bitfinexcom/lib-js-util-math.git",
     "bfx-api-node-core": "^1.5.5",
-    "bfx-api-node-models": "^1.4.1",
+    "bfx-api-node-models": "git+https://github.com/avsek477/bfx-api-node-models.git#add-visible-on-hit-hidden-orders",
```
Comment (avsek477) [inline package.json:43]: needs to be changed after this [PR](https://github.com/bitfinexcom/bfx-api-node-models/pull/77) is merged.

File: package.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
   ],
   "dependencies": {
     "@bitfinex/lib-js-util-math": "git+https://github.com/bitfinexcom/lib-js-util-math.git",
-    "bfx-api-node-core": "^1.5.5",
+    "bfx-api-node-core": "git+https://github.com/avsek477/bfx-api-node-core.git#orders-models-update",
     "bfx-api-node-models": "git+https://github.com/avsek477/bfx-api-node-models.git#add-visible-on-hit-hidden-orders",
```
Comment (avsek477) [inline package.json:43]: needs to be changed after this [PR](https://github.com/bitfinexcom/bfx-api-node-models/pull/77) is merged.

File: lib/accumulate_distribute/util/generate_order.js [hunk 3]
```diff
@@ -52,6 +53,14 @@ const generateOrder = (instance = {}) => {
       meta: { _HF: 1 }
     })
   } else if (orderType === 'LIMIT') {
+    console.log('ORDER------------------------->', new Order({
```
Comment (tarcisiozf) [inline lib/accumulate_distribute/util/generate_order.js:56]: should we keep this console.log or it was just for dev purposes?

**Review by tarcisiozf — APPROVED** [review]
lgtm

File: lib/accumulate_distribute/util/generate_order.js (hunk 3 above)
Comment (avsek477) [reply lib/accumulate_distribute/util/generate_order.js]: for dev purposes, forgot about it.

**Review by tarcisiozf — APPROVED** [review]

**Pull request merged**